pkg/controller/performanceprofile/components/machineconfig/machineconfig.go

```diff
@@ -83,7 +84,11 @@ func New(assetsDir string, profile *performancev1alpha1.PerformanceProfile) (*ma
 		return nil, err
 	}
 
-	mc.Spec.Config = *ignitionConfig
+	rawIgnition, err := json.Marshal(ignitionConfig)
```

Comment:
Why this change?

Reply:
Because of the change under the machine config repository, I updated our deps to use release-4.5 instead of 4.4